Meetings/devel-call-2025-01-29.md

```diff
@@ -0,0 +1,47 @@
+# Meeting Minutes: SVSM Development Call (January 29th, 2025)
+
+## Attendees:
+
+Adam Dunlap, Cláudio Carvalho, Dionna Glaze, Hajira Zaman, Joerg Roedel, Jon Lange, Nicolai Stange, Peter Fang, Tom Lendacky, Vasant Karasulli
+
+## Topics:
+
+### GitHub Organization Membership:
+
+* Joerg Roedel encouraged attendees to become members of the COCONUT-SVSM organization on GitHub to facilitate PR assignments, issue management, and code review requests.
+
+### Doc Website Generation Workflow Failure:
+        
+* Joerg Roedel reported a failing doc website generation workflow on GitHub, possibly related to recent TPM changes.
```

Comment:
@joergroedel  Do we have any issue where tracing this?

I guess you are referring to https://github.com/coconut-svsm/svsm/actions/runs/13014544401/job/36300316231

I fixed the unused var upstream, so I may try to update our submodule removing the usage of `-Wno-error=unused-variable` in our makefile.

Reply:
I think the problem is that we don't install `bindgen-cli` in the workflow, I'm opening a PR.

Reply:
PR opened here: https://github.com/coconut-svsm/svsm/pull/605

Reply:
Thanks for fixing this. I will have a look at the proposed PRs.